File: 3363/3363.cs
public class Solution {
    public int FindTheMaximumNumberOfFruitsCo(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(FindTheMaximumNumberOfFruitsCo(root.left), FindTheMaximumNumberOfFruitsCo(root.right));
    }
}

File: 3363/3363.cpp
class Solution {
public:
    int findTheMaximumNumberOfFruitsCo(TreeNode* root) {
        if(!root) return 0;
        return 1+max(findTheMaximumNumberOfFruitsCo(root->left),findTheMaximumNumberOfFruitsCo(root->right));
    }
};